sdk/identity/azure-identity-broker/README.md

```diff
@@ -47,12 +47,12 @@ Microsoft Entra applications rely on redirect URIs to determine where to send th
 This example demonstrates using `InteractiveBrowserBrokerCredential` as a broker-enabled credential for authenticating with the `BlobServiceClient` from the [azure-storage-blob][azure_storage_blob] library. Here, the `win32gui` module from the `pywin32` package is used to get the current window.
 
 ```python
-import win32gui
```

Comment:
Why not keep the gui-based window handle example for windows and add a console-based example separately, as suggested in my previous comment?